src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -624,10 +625,34 @@ public void setNoDamageTicks(int ticks)
 	}
 
 	@Override
-	public Player getKiller()
-	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+	public void kill() {
```

Comment:
I think it would be a very smart idea to couple `#setKiller(...)` and `#kill()` together.
So that calling `kill(null)` or `kill(player)` would also set the killer variable.
This could just be an overload though and we keep the current two methods.

Reply:
Good Idea, I'll change that

Reply:
I concur with this.
Also:
```suggestion
	public void kill()
	{
```